credsweeper/credentials/candidate.py

```diff
@@ -32,78 +32,20 @@ def __init__(self,
                  config: Config,
                  validations: List[Validation] = None,
                  use_ml: bool = False) -> None:
+        self.line_data_list = line_data_list
+        self.patterns = patterns
+        self.rule_name = rule_name
+        self.severity = severity
+        self.config = config
+        self.validations: List[Validation] = validations if validations is not None else []
+        self.use_ml = use_ml
```

Comment:
It seems all type hints are removed.. Is it helpful on memory reduction?

Reply:
The hints are not necessary here - they are passed from function and MyPy recognizes it. Just code cleanup. 
The hints do not impact on memory reducing.

Reply:
Okay. Let's remove type hints.